pkg/image/definition.go

```diff
@@ -11,12 +11,32 @@ const (
 	TypeRAW = "raw"
 )
 
+const (
+	ArchTypeIntel Arch = "x86_64"
```

Comment:
Suggest this is `ArchTypeX86` rather than `ArchTypeIntel`, I mean, it was AMD that came up with x86_64 anyway ;-)